tests/system/python/conftest.py

```diff
@@ -1367,56 +1367,82 @@ def pytest_configure():
     pytest.PKG_MGR = 'yum' if pytest.IS_REDHAT else 'apt'
 
 
-def restart_and_wait_for_fledge(fledge_url, wait_time, auth_token=None):
+def restart_and_wait_for_fledge(fledge_url, wait_time, auth_token=None, custom_port=None, https_enabled=False):
     """ Restarts the Fledge service and waits until it becomes responsive
 
     Args:
         fledge_url (str): base fledge url
         wait_time (int): Seconds between retries
         auth_token (str): Authorization Token (Optional)
+        custom_port (int, optional): Custom port. Defaults to None.
+        https_enabled (bool, optional): Whether to use HTTPS instead of HTTP. Defaults to False.
     Raises:
         AssertionError: If Fledge failed to restart
 
     Returns:
         JSON Document
     """
+    import ssl
     from contextlib import closing
     headers = {"authorization": auth_token} if auth_token else {}
 
-    with closing(http.client.HTTPConnection(fledge_url)) as connection:
+    if "1995" in fledge_url:
+        ssl_context = ssl._create_unverified_context()
+        connection = http.client.HTTPSConnection(fledge_url, context=ssl_context)
+    else:
+        connection = http.client.HTTPConnection(fledge_url)
+        
+    with closing(connection) as connection:
         connection.request("PUT", '/fledge/restart', headers=headers, body=json.dumps({}))
-        r = connection.getresponse()
-        assert 200 == r.status
-        r = r.read().decode()
-        jdoc = json.loads(r)
-        assert "Fledge restart has been scheduled." == jdoc['message']
+        response = connection.getresponse()
+        assert response.status == 200
+        response_data = response.read().decode()
+        jdoc = json.loads(response_data)
+        assert jdoc['message'] == "Fledge restart has been scheduled."
```

Comment:
We should choose one style: either actual == expected or expected == actual; we should not mix styles.
And I believe most of the files are using with `expected == actual`